tools/src/main/scala/scala/scalanative/codegen/Lower.scala

```diff
@@ -4,6 +4,19 @@ package codegen
 import scala.collection.mutable
 import scalanative.util.{ScopedVar, unsupported}
 import scalanative.nir._
+import scalanative.nir.Rt.{
+  ClassName,
+  ClassIdName,
+  ClassTraitIdName,
+  ClassNameName,
+  ClassSizeName,
+  ClassIdRangeUntilName,
+  StringName,
+  StringValueName,
+  StringOffsetName,
+  StringCountName,
+  StringCachedHashCodeName
+}
```

Comment:
Is such import really needed? Can't we import all of them or use qualified names like we used to do?